cypress/e2e/navigation.cy.ts

```diff
@@ -0,0 +1,33 @@
+describe('Navigation Bar', () => {

```

Comment:
cypress e2e test flows should indicate end-to-end user behaviour... the test you have written is fine for now but it feels like "single component" tests. 

a better flow would be
- navigate to liveclass
- click create
- go on to create
- see if created on home page again

and subtests therein

Reply:
It's just a test flow, will write complete flows soon.